articles/app-service/overview-managed-identity.md

```diff
@@ -472,29 +472,34 @@ To learn more about configuring AzureServiceTokenProvider and the operations it
 
 For Java applications and functions, the simplest way to work with a managed identity is through the [Azure SDK for Java](https://github.com/Azure/azure-sdk-for-java). This section shows you how to get started with the library in your code.
 
-1. Add a reference to the [Azure SDK library](https://mvnrepository.com/artifact/com.microsoft.azure/azure). For Maven projects, you might add this snippet to the `dependencies` section of the project's POM file:
+1. Add a reference to the [Azure SDK library](https://mvnrepository.com/artifact/com.azure.resourcemanager/azure-resourcemanager). For Maven projects, you might add this snippet to the `dependencies` section of the project's POM file:
 
     ```xml
     <dependency>
-        <groupId>com.microsoft.azure</groupId>
-        <artifactId>azure</artifactId>
-        <version>1.23.0</version>
+      <groupId>com.azure.resourcemanager</groupId>
+      <artifactId>azure-resourcemanager</artifactId>
+      <version>2.10.0</version>
     </dependency>
     ```
 
-2. Use the `AppServiceMSICredentials` object for authentication. This example shows how this mechanism may be used for working with Azure Key Vault:
+2. Use the `ManagedIdentityCredential` object for authentication. This example shows how this mechanism may be used for working with Azure Key Vault:
 
     ```java
-    import com.microsoft.azure.AzureEnvironment;
-    import com.microsoft.azure.management.Azure;
-    import com.microsoft.azure.management.keyvault.Vault
+    import com.azure.core.management.AzureEnvironment;
+    import com.azure.core.management.profile.AzureProfile;
+    import com.azure.identity.ManagedIdentityCredential;
+    import com.azure.identity.ManagedIdentityCredentialBuilder;
+    import com.azure.resourcemanager.AzureResourceManager;
+    import com.azure.resourcemanager.keyvault.models.Vault;
     //...
-    Azure azure = Azure.authenticate(new AppServiceMSICredentials(AzureEnvironment.AZURE))
-            .withSubscription(subscriptionId);
-    Vault myKeyVault = azure.vaults().getByResourceGroup(resourceGroup, keyvaultName);
+    AzureProfile azureProfile = new AzureProfile(AzureEnvironment.AZURE);
+    ManagedIdentityCredential managedIdentityCredential = new ManagedIdentityCredentialBuilder().build();
+    AzureResourceManager azure = AzureResourceManager.authenticate(managedIdentityCredential, azureProfile).withSubscription("subscription");
 
-    ```
+    Vault vault = azure.vaults().getByResourceGroup("resourceGroup", "keyVaultName");
 
+    ```
+For more information on how to use the Azure SDK for Java, please refer to this [quickstart guide](https://aka.ms/azsdk/java/mgmt)
```

Comment:
Should we also add a reference for general Azure Java SDK (in additional to mgmt)?
E.g. https://docs.microsoft.com/azure/developer/java/sdk/identity-azure-hosted-auth or https://github.com/Azure/azure-sdk-for-java/wiki/Azure-Identity-Examples#authenticating-a-user-assigned-managed-identity-with-defaultazurecredential

If user want to use MSI within a webapp, they more likely going to use it for data-plane SDK.

Reply:
makes sense!